python/orca/src/bigdl/orca/learn/pytorch/pytorch_pyspark_estimator.py

```diff
@@ -76,7 +76,11 @@ def __len__(self):
                 return get_size(self.y)
 
             def __getitem__(self, i):
-                return index_data(self.x, i), index_data(self.y, i)
+                index_data_x = index_data(self.x, i)
```

Comment:
For we can only allocate x and y as two split part here, we need to reform multi-input as [x1, x2] as the whole x